java/src/jmri/jmrit/logixng/tools/swing/TreeEditor.java

```diff
@@ -1296,13 +1297,13 @@ private boolean parentIsSystem(FemaleSocket femaleSocket) {
      * @return true if not edit system node, else return false
      */
     private boolean abortEditAboutSystem(Base b) {
-        int result = JOptionPane.showConfirmDialog(
+        int result = JmriJOptionPane.showConfirmDialog(
                 this,
                 Bundle.getMessage("TreeEditor_ChangeSystemNode"),
                 b.getLongDescription(),
-                JOptionPane.YES_NO_OPTION);
+                JmriJOptionPane.YES_NO_OPTION);
 
-        return result == JOptionPane.NO_OPTION;
+        return ( result != JmriJOptionPane.YES_OPTION );
```

Comment:
Why this change? I'm not against it, I just want to understand why it's relevant.

Reply:
Thanks.